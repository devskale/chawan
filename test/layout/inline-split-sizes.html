<!DOCTYPE html>
before <span style="background-color: red"> t<br>
<div>carthago delenda</div>
<span style="padding-left: 5em"></span><span>est</span><br>
hello</span> after
