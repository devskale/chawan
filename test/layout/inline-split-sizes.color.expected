before [48;2;255;0;0mt[49m
carthago delenda
[48;2;255;0;0m          est[49m
[48;2;255;0;0mhello[49m after
